targets/ChibiOS/ORGPAL_PALTHREE/common/usbcfg.c

```
@@ -12,297 +12,292 @@ SerialUSBDriver SDU1;
+    {sizeof usb_device_description - sizeof(wchar_t), (uint8_t *)(&usb_device_description)},
+    {sizeof usb_serial_number - sizeof(wchar_t), (uint8_t *)(&usb_serial_number)},
 };
 
-// Convert Hex 32Bits value into char 
+// Convert Hex 32Bits value into char
 // value: value to convert
-// pbuf: pointer to the buffer 
+// pbuf: pointer to the buffer
 // len: buffer length
-void IntToUnicode(uint32_t value , uint8_t *pbuf, uint8_t len)
+void IntToUnicode(uint32_t value, uint8_t *pbuf, uint8_t len)
```

Cppcheck (2.20.0)

[style] The function 'IntToUnicode' should have static linkage since it is not used outside of its translation unit.

(staticFunction)